.github/workflows/connectivity-tests.yaml

```diff
@@ -110,5 +110,5 @@ jobs:
           E2E_TESTS_ATLAS_X509_PEM: ${{ secrets.E2E_TESTS_ATLAS_X509_PEM }}
           # Matches what we are doing in Evergreen
           MONGODB_VERSION: '4'
-        run: COMPASS_RUN_DOCKER_TESTS=true npm run test --workspace mongodb-data-service
+        run: COMPASS_RUN_DOCKER_TESTS=true npm run test-connectivity --workspace mongodb-data-service
```

Comment:
Was this not running the connectivity tests previously or are we accounting for a recent change here?

Reply:
Is a recent improvement `test-connectivity` only runs connectivity tests, before was running all the tests in data-service and the connectivity tests